### DISCUSSION TITLE
Can it be used with a hosted wordpress with limited ssh?

### DISCUSSION THREAD

**Discussion by krasi-georgiev** [flyntwp/flynt#513] [Q&A] [answered]
I am trying to use it with greengeeks where Composer is installed, but npm isn't.

Why is npm even needed?

**Comment by aaronmeder** [comment, marked as answer]
Hey @krasi-georgiev 

Flynt is a starter theme for developers. It's meant to be built upon, not to be used straight away.

NPM is used during development (usually locally on your dev computer) to install and bundle JS & CSS libraries.

When you want to put the theme on your production server you run `npm run build` or `npm run build:production` on your dev computer (or during a deployment pipeline build step). This generates a dist folder which contains your bundled JS / CSS / graphics.
Transfer the theme including that dist folder to your production server - no need for NPM or node_modules folder there.

Hope that helps.
Aaron 👋🏻
